src/jats/__tests__/__snapshots__/jats-importer.test.ts.snap

```diff
@@ -796,7 +796,7 @@ Object {
       "content": Array [
         Object {
           "attrs": Object {
-            "category": "MPSectionCategory:abstract",
+            "category": "",
             "dataTracked": null,
             "id": "MPSection:test",
           },
```

Comment:
@mnatsheh, this is what I meant by my [comment](https://github.com/Atypon-OpenSource/manuscripts-transform/pull/158#issuecomment-2450539295), the sections under the `abstracts` node are added into a section called abstract. for example, this is what the dom-parser receives from [jats-import.xml](https://github.com/Atypon-OpenSource/manuscripts-transform/blob/master/src/jats/__tests__/__fixtures__/jats-import.xml)
```
<sec sec-type="abstracts"><title>Abstracts</title>
  <sec sec-type="abstract"><title>Abstract</title>
     *** other sections, e.g Objectives
```
I'm not sure if that behavior is intended or we need to fix it.